### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Notebook_Toolbar_Usability: The separator color contrasts is below 3:1

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#11951]
["Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11y_AzureDataStudioStayHealthy;#A11yTCS;#A11yUsable;#Benchmark;#DesktopApp;#SQL Azure Data Studio;#FTP;#ColorContrast;#A11ySev4;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.20.0-insider (system setup)
Commit: fd29e35
Date: 2020-07-02T05:40:02.292Z
VS Code: 1.46.0
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Tool: Accessibility insights for desktop

**Repro Steps:**
1.Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Right-click connection and click New Notebook
4. Click on cell and select code cell navigate to toggle more and open it .
5. Launch Accessibility insights for desktop and select third tab contrast.

**Actual:**
When the color contrast of the separator under toggle more button is checked the ratio is 1.557:1

**Expected:**
The color contrast of the separator should be 3:1 or greater

**User Impact:**
Low vision users will face difficulty in viewing the separate sections in the lsit .

**Attachment link for Reference:**
11194A11y_AzureDataStudioStayHealthy_Notebook_Toolbar_ColorContrast
![image](https://user-images.githubusercontent.com/24580570/91173946-3dba7b80-e6fc-11ea-83f8-2b5083b3bf6a.png)

**Comment by chlafreniere** [comment]
Hey @lucyzhang929 would be great to get this in for August. Sound ok?

**Comment by lucyzhang929** [comment]
@chlafreniere Chatted with the accessibility team and since this is a usability bug, it would be good to fix but given the comment with examples of separators in other apps in the PR linked above, I'll move this to backlog.

**Comment by alanrenmsft** [comment]
bulk closing the usability level issues since we are not planning to work on it anytime soon.
